README.md

```diff
@@ -12,19 +12,34 @@ Dataset Insights maintains a pip package for easy installation. It can work in a
 
 ### Dataset Statistics
 
-We provide a sample [notebook](notebooks/SynthDet_Statistics.ipynb) to help you get started with dataset statistics for the [SynthDet](https://github.com/Unity-Technologies/SynthDet) project. We plan to support other sample Unity projects in the future.
+We provide a sample [notebook](notebooks/Perception_Statistics.ipynb) to help you load synthetic datasets generated from the [Perception package](https://github.com/Unity-Technologies/com.unity.perception) and visualize dataset statistics. We plan to support other sample Unity projects in the future.
 
 ### Dataset Download
 
-Dataset download provides tools to download datasets from HTTP(s), GCS and Unity simulation project . You can run `download` command:
+Dataset download provides tools to download datasets from HTTP(s), GCS and Unity simulation project . You can run `download` command from cli or use api to download the dataset.
```

Comment:
This makes it sond like "dataset download is a separate pkg which is confusing". Also missing articles "the command" and "the API"